### ISSUE TITLE
what is the mean of 99998 about student model :model_99998.pt

### ISSUE THREAD

**Issue by muzi2018** [CAI23sbP/Isaaclab_Parkour#6]
when i run the following script:
python scripts/rsl_rl/train.py --task Isaac-Extreme-Parkour-Student-Unitree-Go2-v0 --seed 1 --headless
it start from 34700, so the model you pushed on git is from 34700 steps (which is teacher policy) to train ?
Learning iteration 34700/84700

**Comment by CAI23sbP** [comment]
@muzi2018  No, Delete logs file.
You can train new mode (not pre-trained)
or You just want to play, please follow [READEM.md teacher ver](https://github.com/CAI23sbP/Isaaclab_Parkour?tab=readme-ov-file#21-pretrained-teacher-policy) and [READEM.md student ver ](https://github.com/CAI23sbP/Isaaclab_Parkour?tab=readme-ov-file#31-pretrained-student-policy)

**Comment by Vvdinosaur** [comment]
> [@muzi2018](https://github.com/muzi2018) No, Delete logs file. You can train new mode (not pre-trained) or You just want to play, please follow [READEM.md teacher ver](https://github.com/CAI23sbP/Isaaclab_Parkour?tab=readme-ov-file#21-pretrained-teacher-policy) and [READEM.md student ver ](https://github.com/CAI23sbP/Isaaclab_Parkour?tab=readme-ov-file#31-pretrained-student-policy)

hi, the link to teacher policy ,there is nothing, could you please upload again?
